README.md

```diff
@@ -51,13 +58,16 @@ regular tick interrupts where possible (for all but the boot CPU core).
 
  * Run `make bindeb-pkg`
 
+Now you will have `.deb` packages in the parent directory.
+
 ## Step 3: Install and Boot the New Kernel
 
-The previous step should have created deb packages in the parent directory. Next:
+Next:
 
  * Install the `linux-image` deb with `dpkg -i` (the exact filename will vary).
 
- * Copy `krun/krun-syscall.h` into `/usr/include/x86_64-linux-gnu/sys/krun-syscall.h`.
+ * Copy `krun-syscall.h` (found in the `krun` directory if you are building
```

Comment:
Could you not mention that including header in the packages breaks glibc or whatever and link to that bug report you made for it? so people know the issue if they want to make there own package.

Reply:
Sure.

Reply:
I thought you said you made a bug report for it on i assume there issue tracker that why i asked.

Reply:
This PR addresses this issue:
https://github.com/softdevteam/krun-linux-kernel/issues/17

I've pushed a fix. Is it OK?

Reply:
yes